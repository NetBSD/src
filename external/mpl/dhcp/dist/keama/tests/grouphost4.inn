# group and host declarations config

# subnet4 declaration
subnet 10.5.5.0 netmask 255.255.255.224 {
    range 10.5.5.5 10.5.5.10;
}

# host declaration
host test1 {
    hardware ethernet 00:0B:FD:32:E6:FA;
    fixed-address 10.5.5.1, 10.10.10.10;
}

# group declaration
group "foobar" {
    default-lease-time 1800;
    option domain-search "example.com", "example.org";
    next-server 192.168.0.1;

    # host declarations
    host test2 {
        hardware ethernet 00:07:0E:36:48:19;
        fixed-address 10.5.5.2;
	option domain-name "example.com";
	option domain-search "example.com", "com";
    }

    host test3 {
        hardware fddi 00:07:0E:36:48:19;
        fixed-address 10.10.10.1;
	default-lease-time 3600;
    }
}

subnet 10.10.10.0 netmask 255.255.255.224 { }
