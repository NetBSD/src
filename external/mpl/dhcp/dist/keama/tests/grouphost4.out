{
  # group and host declarations config
  # subnet4 declaration
  /// This configuration declares some subnets but has no interfaces-config
  /// Reference Kea #245
  "Dhcp4": {
    "subnet4": [
      {
        "id": 1,
        "subnet": "10.5.5.0/27",
        "pools": [
          {
            "pool": "10.5.5.5 - 10.5.5.10"
          }
        ],
        "reservations": [
          # host declaration
          {
            "hostname": "test1",
            "hw-address": "00:0b:fd:32:e6:fa",
            "ip-address": "10.5.5.1"
//          "extra-ip-addresses": [
//            "10.10.10.10"
//          ]
          },
          # host declarations
          {
            "hostname": "test2",
            "hw-address": "00:07:0e:36:48:19",
            "ip-address": "10.5.5.2",
            "option-data": [
              {
                "space": "dhcp4",
                "name": "domain-name",
                "code": 15,
                "data": "example.com"
              },
              {
                "space": "dhcp4",
                "name": "domain-search",
                "code": 119,
//              "original-data": "\"example.com\", \"com\"",
                "data": "example.com, com"
              }
            ],
            "next-server": "192.168.0.1"
          }
        ]
      },
      {
        "id": 2,
        "subnet": "10.10.10.0/27",
        "reservations": [
//        {
//          "hostname": "test3",
//          "hw-address": "fddi 00:07:0e:36:48:19",
//          "ip-address": "10.10.10.1",
//          /// default-valid-lifetime in unsupported scope
//          "valid-lifetime": 3600,
//          "option-data": [
//            {
//              "space": "dhcp4",
//              "name": "domain-search",
//              "code": 119,
//              "original-data": "\"example.com\", \"example.org\"",
//              "data": "example.com, example.org"
//            }
//          ],
//          "next-server": "192.168.0.1"
//        }
        ]
      }
    ],
    "host-reservation-identifiers": [
      "hw-address"
    ]
  }
}
